content/general/aws-authentication.textile

```diff
@@ -218,7 +218,7 @@ Create an IAM role as follows:
 
 3. For Account ID specify 203461409171. This is the Ably AWS account.
 
-4. Click the "Require external ID checkbox" and then enter an external ID of @<Your_Ably_Account_ID>.<Your_Ably_app_ID>@. This is also displayed to you when you create an Ably Lambda rule and select the "ARN of an assumable role" radio button in the create rule dialog.
+4. Click the "Require external ID checkbox" and then enter an external ID of @<Your_Ably_Account_ID>.<Your_Ably_app_ID>@. This is also displayed when you create an Ably Lambda, AWS Kinesis, or SQS integration rule and select the "ARN of an assumable role" radio button in the create rule dialog. Learn more about "finding your App ID here":https://faqs.ably.com/how-do-i-find-my-app-id.
```

Comment:
> an Ably Lambda, AWS Kinesis, or SQS integration rule

Let's just tighten this up slightly with "an Ably AWS Lambda, AWS Kinesis, or AWS SQS integration rule."